front/src/applications/operationalStudies/components/SimulationResults/SimulationResultsMap/TrainHoverPosition.tsx

```diff
@@ -38,7 +38,7 @@ function getSpeedAndTimeLabel(isSelectedTrain: boolean, ecoBlocks: boolean, poin
           km/h
         </span>
         <span className="ml-2 small">
-          {point.speedTime && datetime2time(point.speedTime.time as any)}
+          {point.speedTime && datetime2time(point.speedTime.time as unknown as Date)}
```

Comment:
better leave the error than to have a forced incorrect type, until we figure out what it is.

if it needs a Date you can do `new Date(point.speedTime.time)`

Reply:
yes ok
Anyway the trainHoverManagt on map was buggy